dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java

```diff
@@ -23,6 +23,7 @@ public class CollectionEntry implements Serializable {
     private String versionName;
     private List<String> descriptorTypes = new ArrayList<String>();
     private boolean verified = false;
+    private String topic;
```

Comment:
I think you might also need to add `topicSelection` (and maybe `approvedAITopic`) in order to address the AI bubble portion of https://github.com/dockstore/dockstore-ui2/pull/2008#discussion_r1723978574